.circleci/config.yml

```diff
@@ -6,8 +6,63 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Build and Test
+          name: Prepare
+          command: npm install
+      - run:
+          name: Build
+          command: npm run build
+      - run:
+          name: Unit Test (with coverage)
+          command: npm run coverage
+      - run:
+          name: Run Chrome tests
+          command: npm run dev & npm run selenium-chrome
+      - run:
+          name: Run Firefox tests
+          command: npm run dev & npm run selenium-firefox
+      - run:
+          name: Run IE tests
+          command: npm run dev & npm run selenium-ie
+      - run:
+          name: Run Safari tests
+          command: npm run dev & npm run selenium-safari
+      - run:
+          name: Run Edge tests
+          command: npm run dev & npm run selenium-edge
+      - run:
+          name: Prepare demo
+          command: ./scripts/demo.sh
+      - persist_to_workspace:
+          root: .
+          paths: ./demo
+  update-demo:
+    docker:
+      - image: node:8.10.0
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - add_ssh_keys:
+          fingerprints:
+              - "c3:03:93:79:5d:c7:23:b8:4a:4d:5d:41:e3:38:3f:84"
+      - run:
+          name: Prepare
           command: |
-            npm install
-            npm run build
-            npm run coverage
+            npm install -g --silent gh-pages
+            git config user.email "ci-build@wikimedia.org"
+            git config user.name "ci-build"
+      - run:
+          name: Deploy demo
+          command: |
+            gh-pages --dist demo
```

Comment:
@stephanebisson is this deploying the demo app?
I see this is not on master, I think that's why I branched from #4 .